src/main/java/io/unlogged/util/MapStoreUtil.java

```diff
@@ -2,10 +2,15 @@
 
 import io.unlogged.Constants;
 
-public class MapStoreName {
+public class MapStoreUtil {
```

Comment:
this needs a better name since it has a very specific usage

Reply:
refactored the name here